CHANGELOG.md

```diff
@@ -25,6 +25,56 @@
   characters to the console on Windows. Calls to `Stdout.add*()` behave as
   before.
 
+### Strong Mode
```

Comment:
But maybe this is different as these are semantic changes to existing language constructs?